rq/batch.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from typing import List, Optional
+from uuid import uuid4
+
+from redis import Redis
+from redis.client import Pipeline
+
+from .exceptions import NoSuchBatchError
+from .job import Job
+from .utils import as_text
+
+logger = logging.getLogger("rq.job")
+
+
+class Batch:
```

Comment:
We'll need a `Batch.all()` to return all batches in this Redis DB. The implementation should be similar to [Worker.all()](https://github.com/rq/rq/blob/master/rq/worker.py#L227).

Reply:
Committed the all method